src/components/RegisterForm/RegisterForm.tsx

```diff
@@ -32,21 +54,53 @@ export function RegisterForm() {
   const isTablet: boolean = useMediaQuery(theme.breakpoints.up('tablet'));
   const mobileGap = 2;
   const tabletGap = 8;
+
   return (
-    <Formik
+    <Formik<CreateUserType>
       enableReinitialize
       validateOnMount={true}
       validateOnChange={true}
       validateOnBlur={true}
       validate={validate}
-      initialValues={{ email: '', fname: '', lname: '', password: '', phone: '' }}
-      onSubmit={function () {
-        throw new Error('Function not implemented.');
-      }}
+      initialValues={{ email: '', name: '', password: '', surname: '' }}
```

Comment:
We should add `roomNumber` to initialValues, but backend currently doesn't support `roomNUmber`. WIll be fixed in #36